### DISCUSSION TITLE
SQLite不可用

### DISCUSSION THREAD

**Discussion by ea6d031** [bs-community/blessing-skin-server#558] [Q&A]
### Blessing Skin 版本

6.2

### PHP 版本

8.0

### 出现问题时所使用的浏览器

Chrome

### 您正在使用的 Web Server

Nginx

### 您正在使用宝塔吗？

- [ ] 是

### 出现了什么问题？

在安装Blessing Skin Server时，数据库选择为SQLite并填入路径就会发生该问题。
出现错误
错误码：14
详细信息：无法连接至 SQLite 服务器，请检查你的配置。服务器返回的信息：SQLSTATE[HY000] [14] unable to open database file
文件是存在的。上网搜索说要更改文件夹权限，但是试过没什么用。
MySQL安装时一切正常，SQLite就会出现问题。
如果可以更快速的配置就好了。
谢谢。

### 重现步骤

在安装Blessing Skin Server时，数据库选择为SQLite并填入路径就会发生该问题。

**Comment by mochaaP** [comment]
確認 pdo_sqlite 正確安裝並啟用。
